src/bidiTab/mapperTabPage.ts

```diff
@@ -56,21 +56,20 @@ function findOrCreateTypeLogContainer(logType: string): HTMLElement {
   return document.getElementById(containerId)!;
 }
 
-export class MapperTabPage {
-  static generatePage() {
-    // If run not in browser (e.g. unit test), do nothing.
-    if (!globalThis.document?.documentElement) {
-      return;
-    }
-    window.MapperTabPage = MapperTabPage;
```

Comment:
I did not see any harm in removing this assignment.

Reply:
I feel like it must have been needed for smth, let's ask Vladimir :)

Reply:
cc @nechaev-chromium

Reply:
window.MapperTabPage is not used by ChromeDriver.

Reply:
Turns out this was needed. It caused the following issue: https://github.com/GoogleChromeLabs/chromium-bidi/issues/433. I am readding it back on https://github.com/GoogleChromeLabs/chromium-bidi/pull/434